typescript/cli/src/tests/commands/helpers.ts

```diff
@@ -38,28 +38,34 @@ export const E2E_TEST_CONFIGS_PATH = './test-configs';
 export const REGISTRY_PATH = `${E2E_TEST_CONFIGS_PATH}/anvil`;
 export const TEMP_PATH = '/tmp'; // /temp gets removed at the end of all-test.sh
 
+export const ANVIL_ADDRESS = '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266';
 export const ANVIL_KEY =
   '0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80';
 export const ANVIL_DEPLOYER_ADDRESS =
   '0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266';
```

Comment:
Considerign that we already have the `ANVIL_DEPLOYER_ADDRESS` variable that stores the same value, is the `ANVIL_ADDRESS` needed?

Reply:
[`939c6ef` (#6321)](https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/6321/commits/939c6ef53df65cd2a9f3b3de5c571829cee183bd)